Subject: [PATCH] protocols: omit frozen markers for SteamRT4 scanner 1.23.1

The marker is interface-version metadata, not a wire-format field. Keep
the version-one interfaces and the explanatory text unchanged. Older
scanners fail DTD validation under --strict when this attribute is present.

--- a/staging/color-management/color-management-v1.xml
+++ b/staging/color-management/color-management-v1.xml
@@ -1736,7 +1736,7 @@
     </event>
   </interface>
 
-  <interface name="wp_image_description_reference_v1" version="1" frozen="true">
+  <interface name="wp_image_description_reference_v1" version="1">
     <description summary="Reference to an image description">
       This object is a reference to an image description. This interface is
       frozen at version 1 to allow other protocols to create
--- a/staging/ext-image-capture-source/ext-image-capture-source-v1.xml
+++ b/staging/ext-image-capture-source/ext-image-capture-source-v1.xml
@@ -39,7 +39,7 @@
     only be done by creating a new major version of the extension.
   </description>
 
-  <interface name="ext_image_capture_source_v1" version="1" frozen="true">
+  <interface name="ext_image_capture_source_v1" version="1">
     <description summary="opaque image capture source object">
       The image capture source object is an opaque descriptor for a capturable
       resource.  This resource may be any sort of entity from which an image
